Lengnick-Style Model/firm-structure-tests.nlogo

```diff
@@ -1,7 +1,7 @@
-extensions [table rnd]
+extensions [table rnd gis]
 
 to go
-  let test-structure table:from-json-file "Firm-structure.json"
+  let test-structure gis:load-dataset "Firm-structure.json"
```

Comment:
You don't need the gis extension. This can be done with the table extension.

Reply:
Sorry, this is a remnant from when the import JSON file wasn't working for me. I tried doing it with gis:load-dataset instead before I got the table:from-json-file extension file